src/ontology/mondo-edit.obo

```diff
@@ -274336,6 +274336,7 @@ synonym: "autism, susceptibility to, 4" RELATED []
 synonym: "chromosome 15q11-q13 DUPLICATION syndrome" RELATED []
 synonym: "chromosome 15Q11.2 Duplication syndrome" RELATED []
 synonym: "dup(15)(q11q13)" EXACT [Orphanet:238446]
+synonym: "Dup15q" EXACT []
```

Comment:
this need a dbxref - add the PMID to the paper where this was mentioned (in the format PMID:XXXXXXXX (no space after the colon)